bears/general/LineCountBear.py

```diff
@@ -18,24 +18,37 @@ def _get_blank_line_count(self, file):
         return num_blank_lines
 
     def run(self, filename, file, max_lines_per_file: int,
-            exclude_blank_lines: bool=False):
+            exclude_blank_lines: bool=False, min_lines_per_file: int=0):
```

Comment:
we can't do 0 by default, many projects will have `__init__` files empty and I'm not so sure it's a relevant thing for most users